app/alarm/ui/src/main/java/org/phoebus/applications/alarm/ui/annunciator/AnnunciatorController.java

```diff
@@ -130,7 +137,9 @@ private void processMessages()
                     {   // Annunciate if marked as stand out.
                         addToTable.accept(message);
                         if (! muted)
-                            annunciator.speak(message.message);
+                        {
+                            loader.stream().forEach(annunciatorProvider -> annunciatorProvider.get().speak(message));
```

Comment:
Without stepping into this in the debugger, it's unclear if this will load all Annunciator implementations and 'get()', that is, construct them, for each annunciated messages. Would be bad for the FreeTTS annunciator to allocate a voice or the AudioAnnunciator to load all sounds each time one message is handled.
Maybe fetch all annunciators once at startup into a `List<Annunciator>` and then just call `speak()` on each?

Reply:
Set a breakpoint in the constructor of the FreeTTS or Audio annunciator constructor, or add a printout there, and check if it's called just once or each time a message is handled.

Reply:
Sorry, I have fixed the SPI contributions being constructed each time.